dashboard/app/config.go

```diff
@@ -167,6 +167,15 @@ type CoverageConfig struct {
 	// WebGitURI specifies where can we get the kernel file source code directly from AppEngine.
 	// It may be the Git or Gerrit compatible repo.
 	WebGitURI string
+
+	// EmailRegressionsTo species the regressions recipient.
+	// If empty, regression analysis is disabled.
+	EmailRegressionsTo string
```

Comment:
Verify that it's a valid email on start, so that we don't crash after weeks from deployment.